test_runner/src/main/kotlin/ftl/args/IArgs.kt

```diff
@@ -82,3 +83,10 @@ interface IArgs {
         val validArgs: Map<String, List<String>>
     }
 }
+
+val IArgs.logLevel
+    get() = if (outputStyle == OutputStyle.Compact) OutputLogLevel.SIMPLE else OutputLogLevel.DETAILED
+
+fun IArgs.setLogLevel() = also {
+    ftl.log.setLogLevel(logLevel)
+}
```

Comment:
IMHO setSomething needs some parameter, so I would suggest to rename it to `setupLogLevel`

Reply:
done